src/main/java/de/olivermakesco/polyspring/impl/ArmorInfo.java

```diff
@@ -0,0 +1,9 @@
+package de.olivermakesco.polyspring.impl;
+
+import net.minecraft.world.item.equipment.ArmorMaterial;
+import net.minecraft.world.item.equipment.ArmorType;
+
+public interface ArmorInfo {
+    ArmorType getArmorType();
+    ArmorMaterial getArmorMaterial();
```

Comment:
Since these are being ducked, the method name should be prefixed with `polyspring$` to avoid naming collisions